jsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
is this a `vscode` specific file? editor-specific configurations probably do not need to be included in the mainline repository